README.md

```diff
@@ -62,7 +62,7 @@ pip install ibeam
 
 ### Startup
 
-#### Docker Image (Recommended):
+#### Docker Image w/ Env secrets:
```

Comment:
Good suggestion, only that for non-fluent English speakers and non-proficient users I'd expand this to 'with environment variable secrets'